core/src/main/java/org/apache/calcite/sql/fun/SqlRowOperator.java

```diff
@@ -82,7 +82,7 @@ public void unparse(
       SqlCall call,
       int leftPrec,
       int rightPrec) {
-    SqlUtil.unparseFunctionSyntax(this, writer, call);
+    writer.getDialect().unparseRow(writer, call, leftPrec, rightPrec);
   }
```

Comment:
We should not modify the unparse logic of `SqlRowOperator`, instead in `SqlDialect`, match the `ROW` call and unparse it specifically.

Reply:
okay. done